Description: Fix FTBFS "format not a string literal and no format arg"
Author: Anton Gladky <gladky.anton@gmail.com>
Last-Update: 2012-03-20

--- a/src/gplt_x11.c
+++ b/src/gplt_x11.c
@@ -2897,22 +2897,22 @@
 
 		    case PseudoColor:
 			fprintf(stderr, ERROR_NOTICE("PseudoColor"));
-			fprintf(stderr, display_error_text_after);
+			fprintf(stderr,"%s", display_error_text_after);
 			break;
 
 		    case GrayScale:
 			fprintf(stderr, ERROR_NOTICE("GrayScale"));
-			fprintf(stderr, display_error_text_after);
+			fprintf(stderr,"%s", display_error_text_after);
 			break;
 
 		    case StaticColor:
 			fprintf(stderr, ERROR_NOTICE("StaticColor"));
-			fprintf(stderr, display_error_text_after);
+			fprintf(stderr,"%s", display_error_text_after);
 			break;
 
 		    case StaticGray:
 			fprintf(stderr, ERROR_NOTICE("StaticGray"));
-			fprintf(stderr, display_error_text_after);
+			fprintf(stderr,"%s", display_error_text_after);
 			break;
 
 		    case DirectColor:
